src/Execution/ModelsLoader/PaginatedModelsLoader.php

```diff
@@ -84,7 +85,9 @@ protected function loadRelatedModels(EloquentCollection $parents): EloquentColle
         return $relatedModels->unique(
             // Compare all attributes because there might not be a unique primary key
             // or there could be differing pivot attributes.
-            static fn (Model $relatedModel): string => $relatedModel->toJson(),
+            static function (Model $relatedModel): string {
+                return json_encode($relatedModel->getOriginal());
+            },
```

Comment:
I don't see the need for a full closure, a short fn is simpler here

Reply:
Fixed